src/state/actions/annotation.js

```diff
@@ -93,3 +93,17 @@ export function deselectAnnotation(windowId, canvasId, annotationId) {
     type: ActionTypes.DESELECT_ANNOTATION, windowId, canvasId, annotationId,
   };
 }
+
+/**
+ * toggleAnnotationDisplay - action creator
+ *
+ * @param  {String} windowId
+ * @param  {String} canvasId
+ * @param  {String} annotationId
```

Comment:
'canvasId' and 'annotationId' are copypasted

Reply:
Good eye. I've updated the comment.